### DISCUSSION TITLE
Training and Testing Models

### DISCUSSION THREAD

**Discussion by Humaan10** [mrdbourke/pytorch-deep-learning#563] [Q&A] [answered]
Hey so I still can not understand the difference between training and testing models and what does testing models even do? So please can any one tell me. Thanks

**Comment by AS1100K** [comment, marked as answer]
**Training Model** - It means that the model sees a lot of data and analyses the common features for identification which it uses to analyse a object which it didn't saw.

**Testing Model** - After the model is trained, we test it to see weather the model is accurate enough to identify data it didn't saw.

**Comment by code-killer0** [comment]
1. Training a model means teaching it using known data and answers.
2. Testing a model means checking how well it performs on new, unseen data.
3. Training helps the model learn patterns.
4. Testing checks if the model can predict correctly in real-world scenarios.
